try/run_central_instances.py

```diff
@@ -0,0 +1,2341 @@
+    possible_aut_profiles = []
+    for profile_inst in profiles_in_dict:
+        profile_org_name = profile_name(org_vm, profile_inst, short=True)
+        if all_profiles_dict[profile_org_name].type == 'autonomous':
+            possible_aut_profiles.append(profile_inst)
+
+    possible_names = [profile_name(org_vm, inst)
+                      for inst in possible_aut_profiles]
+    print('POSSIBLE_AUTONOMOUS_PROFILES %s: %s' % (nickname,
+                                                   possible_names))
+
+    ref_direction = None
+    try:
+        ref_direction = determine_reference_direction_slow(
+            nickname,
+            server, org_vm,
+            possible_autonomous_profiles=possible_aut_profiles)
+    except Exception as ex:  # pylint: disable=broad-except
+        print('ERROR: determine_reference_direction failed: Exception=%s' % ex)
+        return
+
+    prop_names = ['Antecedent', 'Dependent']
+    if ref_direction == 'snia':
+        prop_names.reverse()
+    print('%s defines CIM_ReferencedProfile using slow method as "%s":\n'
+          '   %s = Referenced/Component Profile,\n'
+          '   %s = Referencing/Autonomous Profile' % (nickname,
+                                                      ref_direction,
+                                                      prop_names[0],
+                                                      prop_names[1]))
+
+    return ref_direction
+
+
+def possible_target_profiles(nickname, server, all_profiles_dict,
+                             org_vm, autonomous=True, output='path'):
+    """
+    Get list of possible autonomous or component profiles based on the list
+    of all profiles and the list of profiles in the defined server.
+
+    Returns list of *paths or insts, or profile names depending on the value
+    of the output parameter.
+    """
+    assert output in ['path', 'name']
+    profiles_in_svr = get_profiles_in_svr(nickname, server,
+                                          all_profiles_dict,
+                                          org_vm)
+    # list of possible autonomous profiles for testing
+    possible_profiles = []
+
+    for profile_inst in profiles_in_svr:
+        profile_org_name = profile_name(org_vm, profile_inst, short=True)
+        if autonomous:
+            if all_profiles_dict[profile_org_name].type == 'autonomous':
+                possible_profiles.append(profile_inst)
+        else:
+            if not all_profiles_dict[profile_org_name].type == 'component':
+                possible_profiles.append(profile_inst)
+
+    if output == 'path':
+        possible_profiles = [inst.path for inst in possible_profiles]
+    elif output == 'name':
+        possible_profiles = [profile_name(org_vm, inst)
+                             for inst in possible_profiles]
+
+    return possible_profiles
+
+
+def test_reference_direction_fast(nickname, server, all_profiles_dict,
+                                  org_vm, st):

```

Comment:
`servers_path` is used as the default. I suggest to name the variable accordingly.

Reply:
This one confusing since we are not using the servers path at all but an instance of ServerDefinition.

DISCUSISON:  What did you mean here?